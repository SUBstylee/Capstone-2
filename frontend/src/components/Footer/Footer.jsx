
import React from 'react'
import { Container, Col, Row } from 'react-bootstrap'

import './Footer.scss';

const Footer = () => {
  return (
    <footer className='Footer'>
      <Container>
        <Row>
          <Col className='text-center'>
            Copyright &copy; Totally Awesome Apparel
          </Col>
        </Row>
        <Row>
          <Col className='text-center'>
            <a className="btn" style={{backgroundColor: '#333333', color:'#fff'}} rel="noopener noreferrer" href="https://github.com/SUBstylee" role="button" target='_blank'><i className="fab fa-github"></i></a>
            <a className="btn" style={{backgroundColor: '#0082ca', color:'#fff'}} rel="noopener noreferrer" href="https://www.linkedin.com/in/jeremy-threlfall/" role="button" target='_blank'><i className="fab fa-linkedin-in"></i></a>
            <a className="btn" style={{backgroundColor: '#06C755', color:'#fff'}} rel="noopener noreferrer" href="https://line.me/R/ti/p/substylee" role="button" target='_blank'><i className="fab fa-line"></i></a>
            <a className="btn" style={{backgroundColor: '#55acee', color:'#fff'}} rel="noopener noreferrer" href="https://twitter.com/jjthrelfall" role="button" target='_blank'><i className="fab fa-twitter"></i></a>
            <a className="btn" style={{backgroundColor: '#3b5998', color:'#fff'}} rel="noopener noreferrer" href="https://www.facebook.com/substylee" role="button" target='_blank'><i className="fab fa-facebook-f"></i></a>
            <a className="btn" style={{backgroundColor: '#ac2bac', color:'#fff'}} rel="noopener noreferrer" href="https://www.instagram.com/jjthrelfall/" role="button" target='_blank'><i className="fab fa-instagram"></i></a>
          </Col>
        </Row>
      </Container>
    </footer>
  )
}

export default Footer