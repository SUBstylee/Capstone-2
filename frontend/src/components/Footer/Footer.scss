.Footer{
    margin: 0 auto;
    position: fixed;
    display: block;
    bottom: 0;
    width: 100%;
    height: 30px;
    background-color: #eeeeee;
}