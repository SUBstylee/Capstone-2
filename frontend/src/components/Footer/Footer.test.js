import { render } from '@testing-library/react';
import Footer from './Footer';

test("renders Footer component without crashing", () => {
    const { getByText } = render(
        <Footer />
    );

    expect(getByText("Copyright Â© Totally Awesome Apparel")).toBeInTheDocument();
});

test("matches snapshot", function () {
    const { asFragment } = render(
        <Footer />
    );
    expect(asFragment()).toMatchSnapshot();
});