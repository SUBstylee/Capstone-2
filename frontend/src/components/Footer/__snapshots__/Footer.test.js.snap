// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="Footer"
  >
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="text-center col"
        >
          Copyright Â© Totally Awesome Apparel
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="text-center col"
        >
          <a
            class="btn"
            href="https://github.com/SUBstylee"
            rel="noopener noreferrer"
            role="button"
            style="background-color: rgb(51, 51, 51); color: rgb(255, 255, 255);"
            target="_blank"
          >
            <i
              class="fab fa-github"
            />
          </a>
          <a
            class="btn"
            href="https://www.linkedin.com/in/jeremy-threlfall/"
            rel="noopener noreferrer"
            role="button"
            style="background-color: rgb(0, 130, 202); color: rgb(255, 255, 255);"
            target="_blank"
          >
            <i
              class="fab fa-linkedin-in"
            />
          </a>
          <a
            class="btn"
            href="https://line.me/R/ti/p/substylee"
            rel="noopener noreferrer"
            role="button"
            style="background-color: rgb(6, 199, 85); color: rgb(255, 255, 255);"
            target="_blank"
          >
            <i
              class="fab fa-line"
            />
          </a>
          <a
            class="btn"
            href="https://twitter.com/jjthrelfall"
            rel="noopener noreferrer"
            role="button"
            style="background-color: rgb(85, 172, 238); color: rgb(255, 255, 255);"
            target="_blank"
          >
            <i
              class="fab fa-twitter"
            />
          </a>
          <a
            class="btn"
            href="https://www.facebook.com/substylee"
            rel="noopener noreferrer"
            role="button"
            style="background-color: rgb(59, 89, 152); color: rgb(255, 255, 255);"
            target="_blank"
          >
            <i
              class="fab fa-facebook-f"
            />
          </a>
          <a
            class="btn"
            href="https://www.instagram.com/jjthrelfall/"
            rel="noopener noreferrer"
            role="button"
            style="background-color: rgb(172, 43, 172); color: rgb(255, 255, 255);"
            target="_blank"
          >
            <i
              class="fab fa-instagram"
            />
          </a>
        </div>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
